rss-reader/Presentation/ChannelFeed/View/Cell/UVFeedTableViewCell.m

```diff
@@ -54,44 +54,45 @@ - (instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSStr
 
 - (void)dealloc
```

Comment:
The '_additionalStack' and '_mainStack' ivars in 'UVFeedTableViewCell' were retained by a synthesized property but not released before '[super dealloc]'

Reply:
`cmd+shift+b` is our friend, fixed